src/jbuild

```diff
@@ -1,8 +1,12 @@
 (jbuild_version 1)
 
-(executable
- ((name       wsproxy)
-  (public_name wsproxy)
-  (libraries   (re.str lwt lwt.unix))
+(executables
+ ((names (wsproxy wsproxy_tests))
```

Comment:
It just need a bit of refactoring of the `jbuild` file. Given that I was trying it, I also made a little reorganization of the folder. You can grab the necessary changes from the last two commits in: https://github.com/mseri/wsproxy/tree/sharad-cp-23210